src/write_prometheus.c

```diff
@@ -756,9 +764,9 @@ static int prom_init() {
 
   if (httpd == NULL) {
     unsigned int flags = MHD_USE_THREAD_PER_CONNECTION;
-#if MHD_VERSION >= 0x00093300
```

Comment:
@rpv-tomsk Thanks! I couldn't figure out why the CI build was failing.  Agreed on the way to rework the PR. Thanks again!